smart-contracts/contracts/Unlock.sol

```diff
@@ -346,6 +366,26 @@ contract Unlock is
     }
   }
 
+  /**
+   * @notice [DEPRECATED] Call to this function has been removed from PublicLock > v9.
+   * @dev [DEPRECATED] Kept for backwards compatibility
+   * This function will keep track of consumed discounts by a given user.
+   * It will also grant discount tokens to the creator who is granting the discount based on the
+   * amount of discount and compensation rate.
+   * This function is invoked by a previously deployed lock only.
+   */
+  function recordConsumedDiscount(
+    uint _discount,
+    uint /* _tokens */
+  )
+    public
+    onlyFromDeployedLock()
+  {
+    // TODO: implement me
+    totalDiscountGranted += _discount;
+    return;
+  }
+
```

Comment:
Actually let's not do that because we don't need to keep track of that for now. (saves gas!)